Microservices/user-service/src/test/java/app/sportahub/userservice/controller/UserControllerTest.java

```diff
@@ -22,14 +24,16 @@
 import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
 
 import java.time.LocalDate;
+import java.util.Arrays;
 import java.util.List;
 
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
 
-@ActiveProfiles("user-service.test")
+import static org.hamcrest.Matchers.*;
+
 @WebMvcTest(UserController.class)
 @Import(TestSecurityConfig.class)
 public class UserControllerTest {
```

Comment:
The tests of all of this class fail. @Ziadsharkos it's ok, simply remove this test file from the PR, and you will add the tests on our next iteration 🔄

Reply:
Agreed!